### DISCUSSION TITLE
"Live" resources

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#466] [Feature requests]
Because of the underlying hotwire implementation, there should be a way to get resources to "live update" on the page when their corresponding model changes in the database. It would be great if Avo provided a callback specification for models that back resources so that rows and show pages would be kept in sync with the server without having to refresh.

**Comment by jony1993** [comment]
Great Idea 👍 
We also have some resources which are more kind of a log.
Would be great if those could be refreshed automatically

**Comment by adrianthedev** [comment]
I started something here https://github.com/avo-hq/avo/pull/889 but I can't seem to make it "easy" to implement. I wrote on the ticket.

Open to suggestions and help on this one.

**Comment by bear-in-mind** [comment]
Would love to help on this too, the need just appeared in my current app for messaging purposes. I'll take a look when I can 💪

**Reply by adrianthedev** [reply]
Cool! Thank you! Check out https://github.com/avo-hq/avo/pull/889, but feel free to start your own PR if you're more comfortable with that.
